### ISSUE TITLE
Create developer project overview, onboarding wiki page

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#109]
Centralize information so new lab members, and eventually external contributors can find everything easily.

Incorporate @Cybernide suggestion to document project board best practices/rules.

**Comment by jeffbl** [comment]
Should be revisited, as what we have now has been built up over time, but needs a high-level review to make sure as more people join over time, we smooth their path to understanding. Some thoughts:

- [ ] Add the W4A paper and video as links for architecture overview

**Comment by Cybernide** [comment]
Should the video links be put onto Youtube, maybe?

**Comment by jeffbl** [comment]
I've just updated the wiki and pinned items in the slack general channel to be relatively up-to-date. The only item above I don't think we've touched on is creating an equipment list. These are almost impossible to keep updated, so I'm loathe to do that, especially since it really would only be the haptic devices, and the "ask @dreamseed87" approach has been successful thus far. @Cybernide ping to you in case you disagree. I've logged new work items for some additional pieces of documentation (#451  #449 #448) that did not get created during our March documentation push. Any futher items should be created as new, separate work items at this point.  I'm still unhappy with the split between gdrive/sharepoint/slack, but I'm not sure we can do much better - there are pointers to all of these things that are referenced via the [new team member starting point on the server wiki](https://github.com/Shared-Reality-Lab/IMAGE-server/wiki), or in the places that points to.
